astropy/table/operations.py

```diff
@@ -129,6 +130,94 @@ def join(left, right, keys=None, join_type='inner',
     return out
 
 
+def setdiff(table1, table2, keys=None):
+    """
+    Take a set difference of table rows.
+
+    The row set difference will contain all rows in table1 that are not
```

Comment:
In this description all the words that refer to parameter names should be RST string literals.  E.g.
```
rows in ``table1`` that are not present in ``table2``.  If the ``keys`` parameter ..
```